OPERATIONS.md

```diff
@@ -0,0 +1,77 @@
+# Operations Manual: logs and alarms
+## Index
```

Comment:
Not sure if a table including "Alarm ID / Severity / Detection strategy / Stop condition / Description /Action" (as the one in https://github.com/telefonicaid/fiware-connectors/blob/develop/flume/doc/operation/alarms.md) is mandatory in this section. Could you check with O&M team, please?